clusterloader2/testing/load/config.yaml

```diff
@@ -160,39 +160,28 @@ steps:
         objectTemplatePath: daemonset-priorityclass.yaml
 
 - name: Starting measurement for waiting for pods
-  measurements:
-  - Identifier: WaitForRunningDeployments
-    Method: WaitForControlledPodsRunning
-    Params:
-      action: start
+  Method: WaitForControlledPodsRunning
```

Comment:
measurements:

missing?

Reply:
Added it. sorry missed it thinking it isn't needed.